.azure-pipelines/ci-build.yml

```diff
@@ -318,119 +325,127 @@ extends:
                   assets: '$(Pipeline.Workspace)\**\*.exe'
                   addChangeLog: false
 
-    - stage: Build_and_deploy_docker_images
-      displayName: 'Build and deploy docker images'
-      condition: or(eq(variables['build.sourceBranch'], 'refs/tags/v'), eq(variables['build.sourceBranch'], variables['PREVIEW_BRANCH']))
-      dependsOn: build
-      pool:
-        name: Azure-Pipelines-1ESPT-ExDShared
-        image: ubuntu-latest
-        os: linux
-      jobs:
-        - job: buildAndPush
-          steps:
-            - task: AzureCLI@2
-              displayName: 'Login to Azure Container Registry'
-              inputs:
-                azureSubscription: 'ACR Images Push Service Connection'
-                scriptType: bash
-                scriptLocation: inlineScript
-                inlineScript: |
-                  az acr login --name msgraphprodregistry
-            
-            - powershell: |
-                $content = [XML](Get-Content ./Directory.Build.props)
-                Write-Host "XML loaded, finding version..."
-                
-                # Handle PropertyGroup as either a single element or array
-                $version = $null
-                if ($content.Project.PropertyGroup -is [array]) {
-                  Write-Host "PropertyGroup is an array, checking each entry..."
-                  foreach ($pg in $content.Project.PropertyGroup) {
-                    if ($pg.Version) {
-                      $version = $pg.Version.ToString().Trim()
-                      Write-Host "Found version in PropertyGroup array: $version"
-                      break
+      - deployment: deploy_docker_image
+        environment: kiota-github-releases
```

Comment:
```suggestion
```
We don't need this environment since we're not writing to github releases here

Reply:
The job fails without an environment variable. I'll create a relevant one on ADO. 
I put it back to test it